content/en/developers/infrastructure/libraries/index.md

```diff
@@ -15,3 +15,19 @@ aliases:
 ---
 
 {{< beta-warning >}}
+
+## Zondax 
+
+Zondax provides the [Filecoin Solidity npm package](#filecoin-solidity) and the [FEVM Precompiles library](#fevm-precompiles).
+
```

Comment:
I'd remove this specific callout to Zondax, and add a simple sentence to the two sections below just saying _"Hey this is made by Zondax, here's a link to their website"_ -- or something to that effect.